src/Microsoft.Maui.Graphics/Microsoft.Maui.Graphics.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.23">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;Xamarin.iOS10;MonoAndroid10.0;Xamarin.Mac20;</TargetFrameworks>
```

Comment:
We need to target iOS, Android and Mac in order to build versions of Maui.Graphics that bundles the native implementations.

Reply:
this is reverted in https://github.com/dotnet/Microsoft.Maui.Graphics/pull/61/commits/e81f04e33684a8a3e30aa2df8a56d5b878bf6e2c

so in the final pull it should be like you wish (and has to be)
otherwise the tests would't have passed, i guess.

Remark:
the general problem is, i develop on a machine without Xamarin.iOS10;MonoAndroid10.0;Xamarin.Mac20 (linux)
so i have to remove that targets, develop, test and finally, before pull, restore the targets again

Reply:
Could I ask that you remove all changes to the Microsoft.MauiGraphics.csproj file, and then I'll merge this in?  The changes to that file are cosmetic only at this point.

Reply:
if you prefer, i cancel this pull request and make a new one out of a patch with only one commit

Reply:
No need for that.  Just remove that one file, and I'll get this in right away.

Reply:
sorry, i think that's not possible (or at least beyond my git knowledge, maybe). unfortunate, the first change is in an combined commit with  correction of Microsoft.Maui.Graphics.Skia.csproj (needs to be NET to compile on linux), so the only choice i have is to revert (what i did).
anyway, i am not happy with the history and finaly code in samples/GraphicsTester.Gtk/MainWindow.cs,
and i've a better implementation of src/Microsoft.Maui.Graphics.Skia.GtkSharp/GtkSkiaDirectRenderer.cs 

so you'll get a shiny (hopefully) new pull request